js/app.js

```diff
@@ -0,0 +1,178 @@
+//Easy Going
+
+// for (i = 1; i <= 20; i++) {
+// 	console.log(i)
+// }
+
+
+//Get Even
+
+// for (i = 0; i <= 200; i += 2) {
+// 	console.log(i)
+// }
+
+
+//Excited Kitten
+
+// const catPhrases = ['...human...why you taking pictures of me?..', '..the catnip made me do it...', '..why does the red dot always get away...'];
+
+// for (i = 0; i < 20; i++) {
+// 	console.log('Love me, pet me! HSSSSSS!')
+// 	if (i % 2 === 0) {
+// 		const randomPhrase = catPhrases[Math.floor(catPhrases.length * Math.random())];
+// 		console.log(randomPhrase);
+// 	}
+// }
+
+//right now this just returns the SAME randomized phrase each time, for all the evens... 
+
+
+//Fizz Buzz
+
+
+// for (i = 1; i < 100; i++) {
+// 		if (i % 3 === 0) {
+// 			console.log('Fizz')
+// 		} if (i % 5 === 0) {
+// 			console.log('Buzz')
+// 		} if (i % 3 === 0 || i % 5 === 0) {
+// 			console.log('Fizzbuzz')
+// 		}
+// }
```

Comment:
This is getting there but it isn't quite right.  Go back and read the question and reread your code slowly, pretending to be like a computer, and think through what you've written. Always be sure to test code as you write it.  Lmk if you want help.
**AND USE `let` in your for loops!!!!**